client/src/leap/soledad/client/crypto.py

```diff
@@ -57,7 +57,7 @@ def _assert_known_encryption_method(method):
         raise crypto.UnknownEncryptionMethodError
 
 
-def encrypt_sym(data, key, method):
+def encrypt_sym(data, key, method=crypto.EncryptionMethods.AES_256_CTR):
```

Comment:
Seems to me that it's always crypto.EncryptionMethods.AES_256_CTR, something else is only used within the tests. (decrypt_sym gets a method from decrypt_doc, which is also the AES_256, but read from the document). Perhaps just remove the parameter here to make it more obvious that AES_256_CTR is the only choice atm? XSALSA20 is only used by tests.